### PR TITLE
Feat/sr 1.5 plus negative rebase

### PR DESCRIPTION
**Pull request by mkurayan**

### CODE REVIEW

File: contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol [hunk 1]
```diff
@@ -42,9 +83,6 @@ struct LimitsList {
     /// @dev Must fit into uint16 (<= 65_535)
     uint256 exitedValidatorsPerDayLimit;
 
-    /// @dev Represented in the Basis Points (100% == 10_000)
-    uint256 oneOffCLBalanceDecreaseBPLimit;
```
[the remark this answers is not in the document]
Comment (mkurayan) [reply contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol:46]: Yeah I agree, but this how it was done in Negative Rebase MR, if you want to change order ideally it should be done in original request.
Here i just merged request, with as less changes as possible.

File: contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol [hunk 2]
```diff
@@ -738,13 +895,25 @@ contract OracleReportSanityChecker is AccessControlEnumerable {
             emit MaxNodeOperatorsPerExtraDataItemCountSet(_newLimitsList.maxNodeOperatorsPerExtraDataItemCount);
         }
         if (_oldLimitsList.requestTimestampMargin != _newLimitsList.requestTimestampMargin) {
-            _checkLimitValue(_newLimitsList.requestTimestampMargin, 0, type(uint64).max);
+            _checkLimitValue(_newLimitsList.requestTimestampMargin, 0, type(uint48).max);
```
[the remark this answers is not in the document]
Comment (mkurayan) [reply contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol:898]: Good catch, thanks!

File: contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol [hunk 3]
```diff
@@ -84,19 +136,28 @@ struct LimitsList {
...
     uint16 exitedValidatorsPerDayLimit;
-    uint16 appearedValidatorsPerDayLimit;
-    uint16 oneOffCLBalanceDecreaseBPLimit;
     uint16 annualBalanceIncreaseBPLimit;
     uint16 simulatedShareRateDeviationBPLimit;
     uint16 maxValidatorExitRequestsPerReport;
     uint16 maxAccountingExtraDataListItemsCount;
     uint16 maxNodeOperatorsPerExtraDataItemCount;
-    uint64 requestTimestampMargin;
+    uint32 requestTimestampMargin;
```
[the remark this answers is not in the document]
Comment (mkurayan) [reply contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol:143]: Removed, thanks!

File: contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkurayan) [reply contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol:46]: After discussion we agreed to update original "Negative rebase limit" MR, `oneOffCLBalanceDecreaseBPLimit` was renamed to `deprecatedOneOffCLBalanceDecreaseBPLimit`. 

In this MR we put new `appearedValidatorsPerDayLimit` on the place of `deprecatedOneOffCLBalanceDecreaseBPLimit` as result we do not break a backward compatibility, all existed fields remained on their original places.

**Review by avsetsin — APPROVED** [review]
lgtm

**Pull request merged**